source/myscript.js

```diff
@@ -1,109 +1,164 @@
+		</div> \
+		<div class='c-thermometer'> \
+			<div class='c-thermometer__trigger'> \
+				<p class='c-thermometer__paragraph'>Was this page helpful?</p> \
 				<div class='c-thermometer__emojis'> \
-					<a href='javascript:void(0);' class='rate-this-page-action' data-rating='Excellent'> \
+					<a href='javascript:void(0);' class='rate-this-page-action' data-rating='Yes'> \
 						<span class='c-thermometer__emoji'>😀</span> \
-						<p>Excellent</p> \
+						<p>Yes</p> \
 					</a> \
-					<a href='javascript:void(0);' class='rate-this-page-action' data-rating='Average'> \
+					<a href='javascript:void(0);' class='rate-this-page-action' data-rating='Somewhat'> \
 						<span class='c-thermometer__emoji'>😐</span> \
-						<p>Average</p> \
+						<p>Somewhat</p> \
 					</a> \
-					<a href='javascript:void(0);' class='rate-this-page-action' data-rating='Poor'> \
+					<a href='javascript:void(0);' class='rate-this-page-action' data-rating='No'> \
 						<span class='c-thermometer__emoji'>🙁</span> \
-						<p>Poor</p> \
+						<p>No</p> \
 					</a> \
 				</div> \
 			</div> \
 		</div>";
 
-	$('body').append(thermometerHtml);
+	$('body footer').append(thermometerHtml);
 
-	$('header .links__icon').on('click', function (){
+	$('header .links__icon').on('click', function () {
 		$("header .header__searchbar").hide();
 		$("header .header__links").slideToggle();
 	});
 
-	$('header .search__icon').on('click', function (){
+	$('header .search__icon').on('click', function () {
 		$("header .header__links").hide();
-		$("header .header__searchbar").slideToggle(400, function() {
-			if($(this).is(':visible')) {
+		$("header .header__searchbar").slideToggle(400, function () {
+			if ($(this).is(':visible')) {
 				$(this).find("input[type=text]").focus();
 			}
 		});
 	});
 
-	if($('.wy-menu-vertical li.current>a').length){
+	if ($('.wy-menu-vertical li.current>a').length) {
 		var sidebarScrollPosition = $('.wy-menu-vertical li.current>a').offset().top;
-		$('.wy-side-scroll').scrollTop(sidebarScrollPosition-120);
+		$('.wy-side-scroll').scrollTop(sidebarScrollPosition - 120);
 	}
 
-	$('body').on('click', '.c-thermometer__emojis a', function(){
-		$(this).parent().hide();
+	$('body').on('click', '.c-thermometer__emojis a', function () {
+		$(this).addClass('selected');
 	});
 
-	$('body').on('click', function(){
+	$('body').on('click', '.c-thermometer-modal__footer .btn', function () {
+		if ($(this).hasClass('btn-link')) {
+			$(this).parents('.c-thermometer-modal__container').fadeOut();
+			$('a.rate-this-page-action').removeClass('selected');
+			$('.c-thermometer-modal__container textarea').val('');
+		} else {
+			var currentString = $(".c-thermometer-modal__container textarea").val()
+
+			if (eventValue > 0) {
+				// Submit DataLayer Event
+				dataLayer.push({
+					event: 'rateThisPage',
+					eventLabel: rating,
+					eventValue: eventValue,
+					eventFeedback: currentString
+				});
+
```

Comment:
```
if (typeof(rudderanalytics) !== 'undefined') {
    rudderanalytics.track("feedback_submitted", { label: rating, rating: eventValue, feedback: currentString});
}
```

Reply:
@asaadmahmood - Can you incorporate @Adovenmuehle's code update please? This update captures the user input provided into Rudderstack. Once this change has been made, we can confirm whether data is moving appropriately.

Reply:
@asaadmahmood I think we probably need both of these calls, not replace the `dataLayer` one. Sorry should've clarified.

Reply:
Done @Adovenmuehle